product_taxes_group/model/product_template.py

```diff
@@ -71,49 +72,28 @@ def fields_view_get(
         return res
 
     # Custom Section
-    def check_coherent_vals(self, ids, vals):
+    def check_coherent_vals(self, vals):
```

Comment:
But please, change the name of the function check_coherent_vals with a more descriptive name. (The aim of my function was to check and return vals, but your function apply changes, doing a write with sudo).